probe/host/system_test.go

```diff
@@ -0,0 +1,38 @@
+package host_test
```

Comment:
Of course, but then you're just mocking and testing for a single platform. It seems like it makes more sense to move the mocking boundary "up" one level, and mock the functions that actually return cross-platform-usable data. Incidentally, that's why I added system_test.go — to make sure the underlying implementations return sane data.